### ISSUE TITLE
HGFS module missing on ubuntu-14.04-amd64 VMware box as built (2.2.9)

### ISSUE THREAD

**Issue by phinze** [chef/bento#673]
Hello! 💟 

I was trying to use the 2.2.9 box hosted here: https://atlas.hashicorp.com/bento/boxes/ubuntu-14.04 tonight for some `kitchen`-ing and I kept hitting errors on the `vmrun enableSharedFolders` step of the Vagrant boot:

```
There was an error mounting the Shared Folders file system inside the guest operating system
```

Poking around the box, it seemed to not have the `vmhgfs` module loaded or available. 🔍 

Rebuilding the box from `master` locally worked AOK. 👌 

Is it possible that the VMware build for `2.2.9` might have had something bad happen that prevented the `vmware.sh` step from completing properly?

**Comment by cheeseplus** [comment]
Version of Fusion/Workstation?

**Comment by tpetchel** [comment]
I just hit the same issue. This used to work on VMWare Fusion Professional Version 7.1.3 (3204469). 

```
config.vm.define "node1-ubuntu" do |cs|
  cs.vm.box = "bento/ubuntu-14.04"
  cs.vm.hostname = "node1-ubuntu"
  cs.vm.network "private_network", ip: "192.168.145.135"
end
```

It broke when I upgraded this morning to VMWare Fusion Professional Version 8.5.0 (4352717). (Similar result as what @phinze reported.)

I could be mistaken, but this seemed relevant: https://github.com/mitchellh/vagrant/issues/6775

As a temporary work-around, the boxcutter box works for me.

```
config.vm.define "node1-ubuntu" do |cs|
  cs.vm.box = "boxcutter/ubuntu1404"
  cs.vm.hostname = "node1-ubuntu"
  cs.vm.network "private_network", ip: "192.168.145.135"
end
```

**Comment by cheeseplus** [comment]
That commit really doesn't have much to do with it as we aren't using open-vm-tools for 14.04 (only 16.04) - anytime the tool versions change in a major way this is likely to happen. I am currently building boxes with Fusion 8.5 and Workstation 12.5 so this should resolve itself with the next version.

**Comment by phinze** [comment]
> anytime the tool versions change in a major way this is likely to happen. I am currently building boxes with Fusion 8.5 and Workstation 12.5 so this should resolve itself with the next version.

Ahh that makes sense. Yeah I'm on `Professional Version 8.5.0 (4352717)`

**Comment by cheeseplus** [comment]
Rolled out 2.3.0 - this should be resolved for Fusion 8.5/Workstation 12.5
